interface/notes_for_s3_interface_deployment.md

```diff
@@ -0,0 +1,18 @@
+# Notes for deploying interface single page app to AWS S3 bucket
+
+Download and configure aws command line interface 
+(http://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html)
+
+Set up users, passwords and permissions with AWS IAM
+(http://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html)
+
+Initialize S3 bucket using AWS console or something like terraform
+(https://www.terraform.io/)
+
+Upload bin folder to S3 bucket using aws-cli 'sync' command
+
+```shell
+# Inside the current folder (adage-server/interface), run
+aws s3 sync ./bin s3://<your s3 bucket name>
+
+```
```

Comment:
@dcgoss is back! Honestly the process of deploying to S3 is so simple that in the time that it takes to discuss whether you want to use S3 it can already be done. If we decide on another option, migrating from S3 is about the same effort as starting from square one anyways.